aten/src/TH/generic/THStorage.cpp

```diff
@@ -20,22 +20,16 @@ size_t THStorage_(elementSize)()
 
 THStorage* THStorage_(new)(void)
 {
-#ifdef THQUANTIZED
-  return THStorage_new(caffe2::TypeMeta::Make<quantized_t>());
-#else
-  return THStorage_new(caffe2::TypeMeta::Make<scalar_t>());
-#endif
+  return THStorage_new();
 }
 
 THStorage* THStorage_(newWithSize)(ptrdiff_t size)
```

Comment:
Need to verify call sites eventually set dtype

Reply:
Call sites for this function look ok to me.